WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -77,6 +77,21 @@
       <%= paginate @posts, param_name: :posts_page, params: { anchor: "posts" } %>
     <% end %>
 
+    <% if !@regulations.empty? %>
```

Comment:
nit: I believe @jonatanklosko changed a bunch of these to `unless`es upstream, so it would be nice to change this one as well for consistency.